targets/android-studio

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+REQUIRES='xiwi gtk-extra'
+DESCRIPTION='Installs Android Studio as a directly launchable xiwi application'
+HOSTBIN='startandroidstudio'
+
+### Append to prepare.sh:
+# Must create the /etc/bash_completion.d directory
+# in order for umake to properly install
+mkdir /etc/bash_completion.d
```

Comment:
I get an error when the directory is pre-existing:
`mkdir: cannot create directory ‘/etc/bash_completion.d’: File exists`

Could/Should a '`|| true`' be appended to the command?